gitpod-io/gitpod
# Use prebuild workspace class for regular workspaces

## Description
Ensure that if a prebuild is used to create a workspace a class is used that is appropriate for the workspace; meaning if the prebuild has a workspace class greater than the currently configured one by the customer the prebuild class is used, otherwise the class configured by the customer is used. 

## Related Issue(s)
Fixes #https://github.com/gitpod-io/gitpod/issues/11972
Needs https://github.com/gitpod-io/ops/pull/3728

## How to test
- Select workspace class g1-large
- Start a prebuild 
- Select workspace class g1-standard
- Create a regular workspace from prebuild
- The workspace should have g1-large 
--
- Start a prebuild for another repository (with g1-standard)
- Select workspace class g1-large
- Create a regular workspace from prebuild
- The workspace should have g1-large

## Release Notes

```release-note
None
```

## Documentation


## Werft options:
